### DISCUSSION TITLE
Add a silent option to match

### DISCUSSION THREAD

**Discussion by Dante1349** [fastlane/fastlane#20718] [Ideas]
When i use the sync_code_signing command, i would like to have a option to get a silenced output, because currently all our certificates are logged in plan text to our jenkins logs. This is security issue in our case, since not everyone that uses the jenkins should be able to see our certificates. I also tried to set verbose to false explicitly, but this didn't do the trick.

**Comment by ManAnRuck** [comment]
I have the same problem in my github action in a public repository. There must be an option for not spending sensitive data 🤔
